generate_fake.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+// generate the fake .well-known folder and upload the folder to the testing web hosting.
+
+use Nexy\NexyCrypt\Authorization\Challenge\Http01Challenge;
+use Nexy\NexyCrypt\Exception\AcmeApiException;
+use Nexy\NexyCrypt\NexyCrypt;
+
+require_once __DIR__.'/vendor/autoload.php';
+
+if ($argc > 2) {
+    echo 'You have to pass too many arguments.'.PHP_EOL;
+    exit(1);
+}
+
+if(isset($argv[1])) {
+    $step = intval($argv[1]);
+} else {
+    echo 'You have to pass the step arguments.'.PHP_EOL;
+    exit(1);
+}
+
+$accounts = json_decode(file_get_contents('tests/ftpserver.json'), true);
+$domain = $accounts['ftpserver'];
+
+// First commented line is for production.
+//$client = new NexyCrypt();
```

Comment:
Commented code should not be here. Plus, I don't think this should be used for production.